way_planner/nodes/way_planner_core.cpp

```diff
@@ -748,6 +753,25 @@ void way_planner_core::PlannerMainLoop()
             }
 #endif
           }
+          else
+          {
+            // If we retried enough, remove the goal from the queue
+            if(newPlanTry >= m_params.planningMaxRetry)
```

Comment:
So, if you set `m_params.planningMaxRetry` to 1, the first run through this function would cause `newPlanTry` to be 1, which means this would evalute to true and produce the warning below. However, this will never have "retried" even though `planningMaxRetry` is 1, which I would assume means that it will "retry" at least once. Thoughts?

Reply:
yes, with 1 it will try it 1 time, are you suggesting a change in the name, like planningMaxAttempt?

Reply:
Yes, I think this is a better name. Please change and I will merge.

Reply:
I did the changes